### DISCUSSION TITLE
LogicFlow案例分享

### DISCUSSION THREAD

**Discussion by wumail** [didi/LogicFlow#1301] [Show and tell]
## 描述
LogicFlow为了提供足够的扩展性，默认情况下并不会做过多的样式、交互上的效果。我们提供了足够多的API支持您去实现您们产品、UI的效果。 在不涉及公司信息安全的情况下，欢迎您把LogicFlow在您们项目中的实际效果分享出来。

## 你可以分享

1. 分享项目中的效果截图。
2. 分享自己用LogicFlow实现的开源项目github地址。
3. 分享使用了LogicFlow的网址。

## 我们还会做什么

1. 在收集到足够多的分享后，我们会从中选区一些用的多的效果做成内置插件。
2. 为后续提供案例展示模块收集素材。

示例站点：https://logic-flow.cn/examples#showcase-bussiness

**Comment by wumail** [comment]
## 一键美化 by [hsole](https://github.com/hsole)

> ![autolayout](https://user-images.githubusercontent.com/42540018/191229478-b057a706-0b41-4641-807c-d9ba354eeefa.gif)
   ![QQ20220920-165532-HD](https://user-images.githubusercontent.com/42540018/191229682-f1d2787c-57b6-4d83-9b31-53fbe2efe5fd.gif)



> https://hsole.github.io/layoutFlow/
   https://github.com/hsole/layoutFlow

**Comment by wumail** [comment]
## 预测LOL S12晋级图 by [hsole](https://github.com/hsole)
> ![image](https://camo.githubusercontent.com/2afee8cddcd9e0fcd297394dca316d980aa1ea6d3da428a37dce25e0eb457320/68747470733a2f2f70392d6a75656a696e2e62797465696d672e636f6d2f746f732d636e2d692d6b3375316662706663702f61376536353536373933653734393239393530663336626338346234356462317e74706c762d6b3375316662706663702d77617465726d61726b2e696d6167653f)
  
> 在线地址：https://hsole.github.io/S12/
    源码地址：https://github.com/hsole/S12

**Comment by wumail** [comment]
## 入门demo学习 by [chenweihan](https://github.com/chenweihan)
> 入门demo学习,请跳转:
  Angular版本LogicFlow的入门DEMO实例请参考
      码云 https://gitee.com/copperpeas/logicflow-angular
      github https://github.com/chenweihan/logicflow-angular
  Vue3LogicFlow的入门DEMO实例请参考((Vue2可参考))
      码云 https://gitee.com/copperpeas/logicflow-vue3
      github https://github.com/chenweihan/logicflow-vue3

**Comment by wumail** [comment]
## vue2 示例 by [Waym1ng](https://github.com/Waym1ng)
> 这是vue2版本，一个简单的demo https://github.com/Waym1ng/logicflow_editing_demo 可以查看json，导入导出，编辑属性的示例

**Comment by eyea** [comment]
使用 `astro +  preact + ci` 的一个例子集合，较为完整的演示了项目开发和自动部署到 `githubpages`

![image](https://github.com/didi/LogicFlow/assets/17020223/0326b31c-4d7f-493e-81e0-21168b1dae84)


[效果预览](https://eyea.github.io/LogicFlowDemos/)
[源码](https://github.com/eyea/LogicFlowDemos)

**Comment by sx1989827** [comment]
公司或组织：阿尔方斯
产品：基于人工智能的团队协作平台Teamlinker，
访问地址：[Teamlinker](https://team-linker.com/) [Teamlinker开源项目](https://github.com/sx1989827/Teamlinker)
截图：

<img width="1540" alt="Screenshot 2023-11-13 at 11 46 24" src="https://github.com/didi/LogicFlow/assets/5820929/f1b7f2d4-df72-4e97-999a-81eb3ed2f470">

**Comment by whki** [comment]
产品：RuleGo-Editor [RuleGo](https://github.com/rulego/rulego) 规则引擎可视化编辑器
RuleGo项目地址：[RuleGo](https://github.com/rulego/rulego) 
演示地址：https://editor.rulego.cc/
截图：
![image](https://github.com/didi/LogicFlow/assets/7507761/3319b594-ac36-4a0e-a47b-91b0fe41c63a)
![image](https://github.com/didi/LogicFlow/assets/7507761/e25ee9d9-b5f6-4e84-9901-4b64a115432f)

**Comment by OSpoon** [comment]
### 自定义插件-拖拽面板：

1. 使用 Quarkc 开发 WebComponent 组件，实现自定义拖转面板：
> 对于一般的前端开发者来说，接触 WebComponent 少可能是这块的一个难点

<img width="971" alt="Snipaste_2024-03-25_14-23-27" src="https://github.com/didi/LogicFlow/assets/10126623/18735238-927d-49c2-a17e-8327d09fd3df">

2. 使用 Ant Design Vue 的 Collapse 组件，实现自定义拖拽面板：
> 对于一般的前端开发者来说，h 函数使用的少可能是这块的一个难点

<img width="978" alt="Snipaste_2024-03-25_14-22-59" src="https://github.com/didi/LogicFlow/assets/10126623/35a96324-4b29-47d7-aa93-ff366e41d79d">

3. 使用 Ant Design React 的 Collapse 组件，实现自定义拖拽面板：
> 对于非的 React 的前端开发者来说，掌握简单的 React 知识即可完成

<img width="951" alt="Snipaste_2024-03-26_11-15-48" src="https://github.com/didi/LogicFlow/assets/10126623/8d0fc8cf-299a-46b1-bba6-25938c9e0106">


### Tip：
源码地址：https://github.com/OSpoon/lf-dnd-panel

**Comment by gaoyakang** [comment]
项目地址：https://github.com/gaoyakang/online_logisim

在线预览：https://mrx.wiki/

![2222222](https://github.com/user-attachments/assets/68912450-b077-4e19-860f-fff6a9f10913)
